scripts/merge_entries.pl

```diff
@@ -0,0 +1,502 @@
+      foreach my $phenotype (@{$merged_phenotypes}) {
+        if($phenotype->get_GenomicFeatureDisease->dbID() == $gfd_to_del) {
+          my $id = $phenotype->dbID();
+          print "  Run: UPDATE genomic_feature_disease_phenotype SET genomic_feature_disease_id = $gfd_to_keep WHERE genomic_feature_disease_phenotype_id = $id\n";
+          # my $sth_pheno = $dbh->prepare($update_gfdpheno);
+          # $sth_pheno->execute($gfd_to_keep, $id) or die $dbh->errstr;
+        }
+      }
+    }
+    
+    if(scalar @{$merged_publications} > 0) {
+      print "\nUpdate publications for GFD_id = $gfd_to_keep\n";
+      my $update_gfdpublication = "UPDATE genomic_feature_disease_publication SET genomic_feature_disease_id = ? WHERE genomic_feature_disease_publication_id = ?";
+      foreach my $publication (@{$merged_publications}) {
+        if($publication->get_GenomicFeatureDisease->dbID() == $gfd_to_del) {
+          my $id = $publication->dbID();
+          print "  Run: UPDATE genomic_feature_disease_publication SET genomic_feature_disease_id = $gfd_to_keep WHERE genomic_feature_disease_publication_id = $id\n";
+          # my $sth_publication = $dbh->prepare($update_gfdpublication);
+          # $sth_publication->execute($gfd_to_keep, $id) or die $dbh->errstr;
+        }
+      }
+    }
+    
+    if(scalar @{$merged_synonyms} > 0) {
+      print "\nUpdate synonyms for GFD_id = $gfd_to_keep\n";
+      my $update_gfd_synonyms = "UPDATE GFD_disease_synonym SET genomic_feature_disease_id = ? WHERE GFD_disease_synonym_id = ?";
+      foreach my $synonyms (@{$merged_synonyms}) {
+        if($synonyms->genomic_feature_disease_id == $gfd_to_del) {
+          my $id = $synonyms->dbID();
+          print "  Run: UPDATE GFD_disease_synonym SET genomic_feature_disease_id = $gfd_to_keep WHERE GFD_disease_synonym_id = $id\n";
+          # my $sth_synonym = $dbh->prepare($update_gfd_synonyms);
+          # $sth_synonym->execute($gfd_to_keep, $id) or die $dbh->errstr;
+        }
+      }
+    }
+    
+    if(scalar @{$merged_panels} > 0) {
+      print "\nUpdate panels for GFD_id = $gfd_to_keep\n";
+      my $update_gfd_panels = "UPDATE genomic_feature_disease_panel SET genomic_feature_disease_id = ? WHERE genomic_feature_disease_panel_id = ?";
+      foreach my $panels (@{$merged_panels}) {
+        if($panels->genomic_feature_disease_id == $gfd_to_del) {
+          my $id = $panels->dbID();
+          print "  Run: UPDATE genomic_feature_disease_panel SET genomic_feature_disease_id = $gfd_to_keep WHERE genomic_feature_disease_panel_id = $id\n";
+          # my $sth_panels = $dbh->prepare($update_gfd_panels);
+          # $sth_panels->execute($gfd_to_keep, $id) or die $dbh->errstr;
+        }
+      }
+    }
+    
+    # Remove the GFD
+    print "\nGoing to delete GFD_id = $gfd_to_del from genomic_feature_disease\n";
+    my $del_gfd = "DELETE FROM genomic_feature_disease WHERE genomic_feature_disease_id = ?";
```

Comment:
Could run the same command as script: https://github.com/EBI-G2P/ensembl-gene2phenotype/blob/master/scripts/populate_panels/delete_gfd_panel.pl